dataplane/standalone/saiserver/acl.go

```diff
@@ -169,6 +169,99 @@ func (a *acl) CreateAclEntry(ctx context.Context, req *saipb.CreateAclEntryReque
 			Masks:   binary.BigEndian.AppendUint64(nil, math.MaxUint64),
 		})
 	}
+	if req.GetFieldAclIpType() != nil {
+		fieldMask := &fwdpb.PacketFieldMaskedBytes{
```

Comment:
which field's mask? Scanning the rest of the code lends me to think this is creating masks depending on what is specified in the request, is that correct?  Perhaps explain how the masks and bytes are being applied to make it clearer what is happening here.